gatsby-config.js

```diff
@@ -27,6 +28,83 @@ module.exports = {
                 includeInDevelopment: false,
             },
         },
+        {
+            resolve: `gatsby-plugin-feed`,
```

Comment:
I just merged a PR which uses the plugins from `gatsby-config` in `gatsby-config.prod.js` to prevent redundancy. This'll be a conflict now.